src/main/java/com/google/api/gax/bundling/BundlingSettings.java

```diff
@@ -103,6 +104,10 @@
   /** Returns the Boolean object to indicate if the bundling is enabled. Default to true */
   public abstract Boolean getIsEnabled();
 
+  /** Get the flow control settings to use. */
+  @Nullable
+  public abstract FlowControlSettings getFlowControlSettings();
```

Comment:
If `FlowControlSettings` has an enabled flag, then this doesn't need to be `@Nullable`.